gatsby-theme-mds/src/components/Rules/DONTs.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { styled } from '@storybook/theming';
+
+const StyledDiv = styled.div(({ color }) => ({
+  borderTopColor: color,
+  borderTopStyle: 'solid',
+  padding: '0 16px'
+}));
+
```

Comment:
Please don't use styled, use our css and css variables. https://innovaccer.github.io/design-system/?path=/story/others-design-tokens-colors--colors